### PR TITLE
Expand `Pkg` guide

### PR DESCRIPTION
**Pull request by 00vareladavid**
This simplifies the existing text and adds more material, in order of increasing complexity.

### CODE REVIEW

**Comment by 00vareladavid** [comment]
How can I preview the HTML rendering?

**Comment by fredrikekre** [comment]
> How can I preview the HTML rendering?

You can just build the docs locally with
https://github.com/JuliaLang/Pkg.jl/blob/c77adfde4067a69f2234573c743783502d6a3c77/.travis.yml#L43-L45

**Comment by 00vareladavid** [comment]
All done. Would anyone like to review?

File: docs/src/getting-started.md [hunk 1]
```diff
@@ -1,104 +1,174 @@
...
 !!! note
-    There are other ways of interacting with Pkg:
-    1. The `pkg` string macro, which is available after `using Pkg`. The command
-       `pkg"cmd"` is equivalent to executing `cmd` in the Pkg REPL.
-    2. The API-mode, which is recommended for non-interactive
-       environments. The API mode is documented in full in the API Reference section
-       of the Pkg documentation.
+    This guide relies on the Pkg REPL to execute Pkg commands.
+    For non-interactive use, we recommend the Pkg API.
+    The Pkg API is fully documented in the API Reference section of the Pkg documentation.
```
Comment (fredrikekre) [inline docs/src/getting-started.md]:
```suggestion
    The Pkg API is fully documented in the [API Reference](@ref) section of the Pkg documentation.
```

File: docs/src/getting-started.md [hunk 2]
```diff
@@ -1,104 +1,174 @@
...
 
+Upon entering the Pkg REPL, you should see a similar prompt:
 
-To add a package, use the `add` command
+    (v1.1) pkg>
```
Comment (fredrikekre) [inline docs/src/getting-started.md]: For consistency with the rest of the manual I would prefer using fenced blocks like before.

File: docs/src/getting-started.md [hunk 3]
```diff
@@ -1,104 +1,174 @@
...
+To add a package, use `add`:
 
-Multiple packages may be specified:
+    (v1.1) pkg> add Example
 
-```
-(v1.1) pkg> add Example JSON StaticArrays
-```
+!!! note
+    Some Pkg feedback has been omitted in order to keep this guide focused.
```
Comment (fredrikekre) [inline docs/src/getting-started.md]:
```suggestion
    Some Pkg output has been omitted in order to keep this guide focused.
```

File: docs/src/getting-started.md [hunk 4]
```diff
@@ -1,104 +1,174 @@
...
+
+Use `update` to update an installed package:
+
+    (v1.1) pkg> update Example
+
+To update all installed packages, use `update` without any arguments:
+
+    (v1.1) pkg> update
+
+## Changing Environments
```
Comment (fredrikekre) [inline docs/src/getting-started.md]:
```suggestion
## Getting Started with Environments
```

File: docs/src/getting-started.md [hunk 5]
```diff
@@ -1,104 +1,174 @@
...
+
+Up to this point, we have covered basic package management: adding, updating and removing packages.
+This will be familiar if you have used other package managers.
+Pkg offers significant advantages over traditional package managers
+  by organizing dependencies into **environments**.
```
Comment (fredrikekre) [inline docs/src/getting-started.md]:
```suggestion
by organizing dependencies into **environments**.
```

File: docs/src/getting-started.md [hunk 6]
```diff
@@ -1,104 +1,174 @@
...
+
+You may have noticed the `(v1.1)` in the REPL prompt.
+This lets us know `v1.1` is the **active environment**.
+There is a single active environment at any point in time.
```
Comment (fredrikekre) [inline docs/src/getting-started.md]:
```suggestion
The active environment is the environment that will be modified by Pkg commands such as `add`, `rm` and `update`.
```

File: docs/src/getting-started.md [hunk 7]
```diff
@@ -1,104 +1,174 @@
...
+
+Lets set up a new environment so we may experiment.
+To set the active environment, use `activate`:
+
+    (v1.1) pkg> activate tutorial
+    [ Info: activating new environment at /tmp/tutorial.
```
Comment (fredrikekre) [inline docs/src/getting-started.md]: This output changed with the merged PR earlier.

File: docs/src/getting-started.md [hunk 8]
```diff
@@ -1,104 +1,174 @@
...
+Julia will load the source code contained in `tutorial/dev/Example`.
 
-To remove this package, use `rm` and specify the package by name (not URL!):
+Lets try it out.
+First we modify the file at `tutorial/dev/Example/src/Example.jl` and add a simple function:
 
-```
-(v1.1) pkg> rm Example
-```
+    plusone(x::Int) = x + 1
```
Comment (fredrikekre) [inline docs/src/getting-started.md]: This can even be a julia code block.

File: docs/src/getting-started.md (hunk 2 above)
Comment (00vareladavid) [reply docs/src/getting-started.md]: Fair point. I'll change it back :+1:

**Comment by 00vareladavid** [comment]
Much better, I think :grin:

**Pull request merged**